app/Library/UserService.php

```diff
@@ -151,4 +161,17 @@ public function getDeptInstructors(string $deptId, array $options = []): Collect
 
         return $instructors;
     }
+
+    public function isUserInstructorInDept(User $maybeInstructor, int $deptId): bool {
+        $instructors = $this->getDeptInstructors($deptId);
+        return $instructors->contains('id', $maybeInstructor->id);
+    }
+
+    public function doesUserManageAnyGroupWithInstructor(User $manager, User $instructor): bool {
+        $userManagedDeptIds = $manager->getManagedGroups()->pluck('dept_id');
+
+        return $userManagedDeptIds->some(function ($deptId) use ($instructor) {
```

Comment:
could I convince you to change this to `contains`? `some` is an alias and it never makes sense to me to read it like that